ComputeClient/Compute.Contracts/Snapshot/Snapshot.cs

```diff
@@ -2434,4 +2436,19 @@ public bool replica
             this.replicaField = value;
         }
     }
-}
\ No newline at end of file
+
+    /// <remarks/>
+    [System.Xml.Serialization.XmlAttributeAttribute()]
+    public bool archived
```

Comment:
Convention seems to be to name these fields like archiveSpecified - all apart from the replica one anyway.

Reply:
A convention that's loosely followed. I'm happy with following the replicaField pattern.